README.md

```diff
@@ -3,12 +3,50 @@
 * Cron Scheduler Microservice.
 
 ## Setup
+With [golang](https://golang.org/doc/install) installed
+
+Install the dependencies listed in `glide.yml` with [glide](https://github.com/Masterminds/glide)
+```
+glide install
+```
 
 Env vars
 ```bash
-export CRON_SRV_DB="host=localhost dbname=cron_srv_dev sslmode=disable"
+export CRON_SRV_DB="postgresql://postgres@localhost/cron_srv_dev?sslmode=disable"
 export CRON_SRV_PORT=3000
 ```
+> **Note:** You must have created the database 'cron_srv_dev' in postgres running at localhost (or replace with valid database name and IP);
+
+## Installation
+```sh
+mkdir -p $GOPATH/src/github.com/EmpregoLigado
+cd $GOPATH/src/github.com/EmpregoLigado 
+git clone https://github.com/EmpregoLigado/cron-srv.git
+cd cron-srv
+go build
+```
+## Running server
+```
+./cron-srv
+# => Starting Cron Service at port 3000
+```
+### Examples
+  - with server running, access the url [localhost:3000/v1/crons](http://localhost:3000/v1/crons) and you should see a empty list `[]`
+  - now we are going to insert some crons, execute in terminal:
+```
+curl -X POST -H "Content-Type: application/json" -d '{"url":"example.com", "expression": "just-test", "status": "active", "max_retries": 2, "retry_timeout": 3}' http://localhost:3000/v1/cron
```

Comment:
@aalvesjr mate great work ❤️ 

Maybe a option to put a real cron expression on example instead of just-test: `* * * * * *`

Also, as we are using this [lib](https://godoc.org/github.com/robfig/cron) we could copy the cron [section](https://godoc.org/github.com/robfig/cron#hdr-CRON_Expression_Format) or reference that somewhere on readme

@aalvesjr wdyt?